scipy/stats/_multivariate.py

```diff
@@ -293,8 +296,11 @@ class multivariate_normal_gen(multi_rv_generic):
     -----
     %(_mvn_doc_callparams_note)s
 
-    The covariance matrix `cov` must be a (symmetric) positive
-    semi-definite matrix. The determinant and inverse of `cov` are computed
+    The covariance matrix `cov` must be a symmetric positive semidefinite
```

Comment:
I would suggest adding the note in the description of the `cov` parameter. See https://github.com/scipy/scipy/pull/15579#issuecomment-1039329817 and https://github.com/scipy/scipy/pull/15579#issuecomment-1039340784

Reply:
Done! Thanks. The string is defined elsewhere so I didn't notice it. I searched and found that it should also be changed in the `multivariate_normal_frozen`.